conf/modules/align_and_call_MT.config

```diff
@@ -52,6 +52,15 @@ process {
         ext.prefix = { "${meta.id}_sorted" }
     }
 
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT:MT_DELETION' {
+        ext.args = '-s --insert-size 16000'
+        publishDir = [
+            path: { "${params.outdir}/mt_sv" },
```

Comment:
maybe we should call this `mt_del`? Do we publish anu of the other MT results in one of the subdirectories? I guess it could go there in that case.

Reply:
I wanted to make mt_sv in order to add eklipse results here too, the only other subdirectory for mt is annotate_mt